src/core/Utils.mjs

```diff
@@ -1119,6 +1141,13 @@ class Utils {
         }[token];
     }
 
+    /**
+     * Check if code is running in a Node environment
+     */
+    static isNode() {
```

Comment:
Perhaps the solution is to provide this and the other environment detection functions as named exports from this file, but not as part of the Utils object, so the whole thing doesn't have to be pulled in wherever they are used. Then you could do:

```javascript
import {isNode} from "./Utils";
```

Reply:
I've stuck with `ENVIRONMENT_IS_*` functions for now.

Reply:
Theyre defined in so many places though, so i might try and reduce that or put them into Utils